keyboards/skyloong/gk75/q1/config.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 JZ-Skyloong (@JZ-Skyloong)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define ENCODER_MAP_KEY_DELAY 20
+
+#define DRIVER_ADDR_2 0b0100011
```

Comment:
No driver 1 address?

Reply:
Driver 1 address is the default address of IS31FL3743A: 0b0100000.